lib/splitclient-rb/engine/api/json_localhost/splits.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+module SplitIoClient
+  module Api
+    # Retrieves split definitions from the Split Backend
```

Comment:
🤔 🤔

Reply:
This class is used in the engine in the same way the Api classes used (for http and redis synching), that is why I added it here.